app/workers/worker_base.rb

```diff
@@ -0,0 +1,7 @@
+class WorkerBase
```

Comment:
I don't see a big advantage in this, in fact I think putting the code in the initializers directory is not great. In general I think that monkey patching should be a last resort and avoided if at all possible. It makes code more complex.

we could avoid monkey patching and still make it a module e.g. 

```
module SidekiqWorker
  def self.included(base)
    base.include(Sidekiq::Worker)
    base.extend(ClassMethods)
  end

  module ClassMethods
    def perform_async_in_queue(queue, *args)
      client_push('class' => self, 'args' => args, 'queue' => queue)
    end
  end
end
```

however, I still don't see big advantages to this over creating a base class for workers.